v2/contribute/decisions/session/0021-handshake-after-jwt-access-token.mdx

```diff
@@ -32,7 +32,9 @@ We can drop the handshake:
 - Anti-csrf mode can be defined on the backend
 - Access token blacklisting will be moved to the backend
 - Validities are already sent in the create and refresh core API responses
-- The core JWKs endpoint can return the necessary public keys
+- The core JWKs endpoint can return the necessary public keys. 
+- Since these JWTs (the access token) should still be signed in the Core, the validation should fetch the public keys from the Core as well, which should also conform to the standard.
```

Comment:
```suggestion
- Since these JWTs (the access token) should still be signed in the Core, the validation should fetch the public keys from the Core as well, which should also conform to the standard. This implies that this route on the core should not expect the apiKey.
```